### DISCUSSION TITLE
Problème sur replay TF1

### DISCUSSION THREAD

**Discussion by bastiengds1** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1252] [Q&A]
Bonjour,
Les replay sur TF1 sont limités à 7 jours. 
Est-il possible de trouver une solution sans avoir un compte tf1 max ? 
Je précise que mon extension est installé sur kodi 20.2

**Comment by joaopa00** [comment]
Non il n'y aucune solution.
